FogCreek/Glitch-Community
# Noisy scale: sign-in pop

## Links
* Remix link: https://noisy-scale.glitch.me/

## GIF/Screenshots
![Screen Shot 2019-05-17 at 3 34 43 PM](https://user-images.githubusercontent.com/938722/57951853-590fc780-78b9-11e9-9525-cf86c8ae1d56.png)
![Screen Shot 2019-05-17 at 3 34 49 PM](https://user-images.githubusercontent.com/938722/57951854-590fc780-78b9-11e9-8af9-87e7ef9d3664.png)
![Screen Shot 2019-05-17 at 3 34 55 PM](https://user-images.githubusercontent.com/938722/57951855-590fc780-78b9-11e9-85a2-df3844e8fb2d.png)
![Screen Shot 2019-05-20 at 3 17 43 PM](https://user-images.githubusercontent.com/938722/58046140-7be8e880-7b12-11e9-8246-5b77782b7c06.png)

## Changes
* modularize CSS for sign-in-pop
* hookify email & sign-in code
* create MultiPopover 

## How To Test:
* visit https://noisy-scale.glitch.me/
* clicking the sign in button in the header should open the sign in menu
* clicking 'sign in with email' should open the email form
* clicking 'use a sign in code' should open the sign-in code form
